src/kit/Card.tsx

```diff
@@ -37,7 +38,12 @@ const Card: Card = ({
   onDropdown,
   size = 'small',
 }: CardProps) => {
-  const classnames = [css.cardBase];
+  const { ui } = useUI();
+  const themeStyle = useMemo(
+    () => (ui.darkLight === DarkLight.Dark ? css.dark : css.light),
+    [ui.darkLight],
+  );
```

Comment:
nit: i think in this case useMemo probably adds more overhead than just doing `const themeStyle = ui.darkLight === DarkLight.Dark ? css.dark : css.light;`